From aa866d33c82a644ccf7965d84289fea819765131 Mon Sep 17 00:00:00 2001
From: Jaroslav Kysela <jkysela@redhat.com>
Date: Wed, 3 Sep 2014 07:40:05 -0400
Subject: [alsa] hda: add PCI IDs for Intel Braswell

Message-id: <140972823573-406-patchset-jkysela@redhat.com>
Patchwork-id: 91102
O-Subject: [RHEL 7.1 PATCH 405/442] ALSA: hda - add PCI IDs for Intel Braswell
Bugzilla: 1112200
RH-Acked-by: John Linville <linville@redhat.com>
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Ivan Vecera <ivecera@redhat.com>

Add HD Audio Device PCI ID for the Intel Braswell platform.
It is an HDA Intel PCH controller.

AZX_DCAPS_ALIGN_BUFSIZE is not necessary for this controller.

Signed-off-by: Libin Yang <libin.yang@intel.com>
Signed-off-by: Takashi Iwai <tiwai@suse.de>

(picked from kernel: f31b2ffcad2b8c57cee5ffc634928bcbc8c6a558)
rhbz#1112200
Signed-off-by: Jaroslav Kysela <jkysela@redhat.com>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/sound/pci/hda/hda_intel.c b/sound/pci/hda/hda_intel.c
index 939dda3..390ef33 100644
--- a/sound/pci/hda/hda_intel.c
+++ b/sound/pci/hda/hda_intel.c
@@ -2017,6 +2017,9 @@ static const struct pci_device_id azx_ids[] = {
  /* BayTrail */
  { PCI_DEVICE(0x8086, 0x0f04),
    .driver_data = AZX_DRIVER_PCH | AZX_DCAPS_INTEL_PCH_NOPM },
+ /* Braswell */
+ { PCI_DEVICE(0x8086, 0x2284),
+   .driver_data = AZX_DRIVER_PCH | AZX_DCAPS_INTEL_PCH },
  /* ICH */
  { PCI_DEVICE(0x8086, 0x2668),
    .driver_data = AZX_DRIVER_ICH | AZX_DCAPS_OLD_SSYNC |
-- 
1.7.1